packages/pds-ember/app/styles/pds/core/typography/stories/index.stories.js

```diff
@@ -0,0 +1,217 @@
+        so that I can prevent the predictable WIP features over the many
+        impediments. It was discovered that by efficiently building the WIP
+        metrics, we can refactor the sustainable MVP code smell across the
+        sustainable lifecycle management. We must perfectly inspect the minimum
+        PR technical debt! As a user, I should be able to detail the lean voice
+        of the customer so that I can document the aggressive VOC bottlenecks
+        behind the domain customers.  Try to document the many velocity, maybe
+        it will help get the XP stakeholders.
+      </p>
+    </section>
+
+    <section>
+      <header>
+        <h4 class="pds-heading pds--3">Heading 3</h4>
+        <p class="pds--body-sm">
+          H3 Section Description:
+          The quick brown fox jumps over the lazy dog.
+        </p>
+      </header>
+
+      <p>
+        H3 Section Copy:
+        As a user, I should be able to block the business lifecycle management
+        so that I can prevent the predictable WIP features over the many
+        impediments. It was discovered that by efficiently building the WIP
+        metrics, we can refactor the sustainable MVP code smell across the
+        sustainable lifecycle management. We must perfectly inspect the minimum
+        PR technical debt! As a user, I should be able to detail the lean voice
+        of the customer so that I can document the aggressive VOC bottlenecks
+        behind the domain customers.  Try to document the many velocity, maybe
+        it will help get the XP stakeholders.
+      </p>
+    </section>
+
+    <section>
+      <header>
+        <h5 class="pds-heading pds--4">Heading 4</h5>
+        <p class="pds--body-sm">
+          H4 Section Description:
+          The quick brown fox jumps over the lazy dog.
+        </p>
+      </header>
+
+      <p>
+        H4 Section Copy:
+        As a user, I should be able to block the business lifecycle management
+        so that I can prevent the predictable WIP features over the many
+        impediments. It was discovered that by efficiently building the WIP
+        metrics, we can refactor the sustainable MVP code smell across the
+        sustainable lifecycle management. We must perfectly inspect the minimum
+        PR technical debt! As a user, I should be able to detail the lean voice
+        of the customer so that I can document the aggressive VOC bottlenecks
+        behind the domain customers.  Try to document the many velocity, maybe
+        it will help get the XP stakeholders.
+      </p>
+    </section>
+  `,
+});
+
+const Fonts = () => ({
+  template: hbs`
+    {{#each (array 'sans-serif' 'monospace') as |family|}}
+      {{#each (array 100 200 300 400 500 600 700 800 900) as |weight|}}
+        <p class="docs-font {{family}} weight-{{weight}}">
+          {{weight}}: {{family}}
+        </p>
+      {{/each}}
+      <br />
+    {{/each}}
```

Comment:
Used in conjunction with `tests/dummy/app/styles/app.scss` changes to demonstrate all weights of all font families.